src/targets/node/nodeAttacherBase.ts

```diff
@@ -39,6 +39,7 @@ export abstract class NodeAttacherBase<T extends AnyNodeConfiguration> extends N
       initialized: async () => {
         toDispose = this.onFirstInitialize(cdp, run);
         await toDispose;
+        return undefined;
```

Comment:
unrelated?

Reply:
This is because I changed the signature of NodeTarget initialized method to:   
`initialized?(target: NodeTarget): Promise<string | undefined>;`

Reply:
This is because I changed the signature of NodeTarget initialized method to:   
`initialized?(target: NodeTarget): Promise<string | undefined>;`